src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -444,6 +452,52 @@ private void SetTransferMode(TransferMode transferMode)
             _httpClient.Dispose();
         }
 
+        public IDisposable OnReceived(Func<byte[], object, Task> callback, object state)
+        {
+            var callBack = new ReceiveCallback(callback, state);
```

Comment:
you need to rename this variable - you have two different variables which differ just by casing in one place. Call it `receiveCallback` or something to that effect.